bool_slice.go

```diff
@@ -0,0 +1,125 @@
+package pflag
+
+import (
+	"strconv"
+	"strings"
+)
+
+// -- boolSlice Value
+type boolSliceValue struct {
+	value   *[]bool
+	changed bool
+}
+
+func newBoolSliceValue(val []bool, p *[]bool) *boolSliceValue {
+	bsv := new(boolSliceValue)
+	bsv.value = p
+	*bsv.value = val
+	return bsv
+}
+
+func (s *boolSliceValue) Set(val string) error {
+	ss := strings.Split(val, ",")
```

Comment:
Yes, that is how a intSlice is handled. 

I'll be glad to add that flag to a test case, and experiment with csv parser to see how it goes. I'll work on it tonight and update the pull request by tomorrow.